### PR TITLE
Extract methods cli

### PR DESCRIPTION
**Pull request by kbrock**
extracted from https://github.com/ManageIQ/floe/pull/223

`Cli#run` method is a bit complicated, so extracting these methods to trim it down

### CODE REVIEW

File: lib/floe/cli.rb [hunk 1]
```diff
@@ -82,5 +76,20 @@ def parse_options!(args)
...
       return workflows_inputs, opts
     end
+
+    def create_credentials(opts)
+      if opts[:credentials_given]
+        opts[:credentials] == "-" ? $stdin.read : opts[:credentials]
+      elsif opts[:credentials_file_given]
+        File.read(opts[:credentials_file])
+      else
+        {}
```
Comment (agrare) [inline lib/floe/cli.rb:86]: @kbrock this looks like a functional change (nil vs {} when credentials not passed).  Was this intentional?  Not a big deal since in Context we do `@credentials = credentials || {}` anyway just curious why the change

Comment (kbrock) [reply lib/floe/cli.rb:86]: I'm overly sensitive right now to us making conversions (e.g.: `making JSON.parse(x) if x.kind_of?(String)` )

I'll undo this. Though I kinda like it better this way.

**Review by agrare — APPROVED** [review]
